NewHorizons/Builder/Props/DetailBuilder.cs

```diff
@@ -52,24 +52,16 @@ public static GameObject GetSpawnedGameObjectByDetailInfo(DetailInfo detail)
         /// <summary>
         /// Create a detail using an asset bundle or a path in the scene hierarchy of the item to copy.
         /// </summary>
-        public static GameObject Make(GameObject go, Sector sector, IModBehaviour mod, DetailInfo detail)
```

Comment:
why the rename? should the other Make's be renamed?

Reply:
By adding the mod param it made it identical in signature to another overload, so I had to merge them one way or another. I decided to rename the params on the shorter one so there'd be fewer overall changes.